manifests/u/UniversityofChicago/StandardMLofNewJersey/110.99.2/UniversityofChicago.StandardMLofNewJersey.installer.yaml

```diff
@@ -0,0 +1,16 @@
+# Created using wingetcreate 0.4.4.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.0.0.schema.json
+
+PackageIdentifier: UniversityofChicago.StandardMLofNewJersey
+PackageVersion: 110.99.2

```

Comment:
```suggestion
PackageVersion: 0.0.0.0
```

Reply:
I've reached out to the SMLNJ maintainers pointing out where in their build they overwrite the version string in the MSI with zeroes, and I've received a response back saying they'll look into it.  I'm going to hold off on fixing the YAML files and instead see if they can release a version where the PackageVersion actually reflects the number (otherwise updates through WinGet aren't going to work properly).